src/main/java/org/javacord/bot/Main.java

```diff
@@ -97,7 +69,10 @@ private static void setupLogging() throws IOException {
             }
         }
 
-        Thread.setDefaultUncaughtExceptionHandler(ExceptionLogger.getUncaughtExceptionHandler());
+        setDefaultUncaughtExceptionHandler((thread, throwable) -> LogManager
+                .getLogger()
+                .atError()
+                .withThrowable(throwable)
+                .log("Caught unhandled exception on thread '{}'!", thread::getName));
```

Comment:
The intended difference is, that the `ExceptionLogger` variant always uses the  `org.javacord.core.util.logging.ExceptionLoggerDelegateImpl` logger and this variant should use the logger according to the class where the error happened.
Unfortunately it was erroneous and always used the `org.javacord.bot.Main` logger.
But I improved it now so that the proper logger is used.